README.md

```diff
@@ -100,6 +100,20 @@ $ rubycritic --help
 | `--deduplicate-symlinks` | De-duplicate symlinks based on their final target     |
 | `--suppress-ratings`     | Suppress letter ratings                               |
 
+
+Rake Task
+---------
+If you want to integrate `rubycritic` into your `Rakefile` just include the below snippet
```

Comment:
This spells out the command line you use, I can change `rubycritic` to RubyCritic